kanidmd/idm/src/idm/credupdatesession.rs

```diff
@@ -587,16 +595,16 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
         self.create_credupdate_session(session_id, Some(intent_id), account, ct)
     }
 
+
     pub fn init_credential_update(
         &mut self,
         event: &InitCredentialUpdateEvent,
         ct: Duration,
     ) -> Result<(CredentialUpdateSessionToken, CredentialUpdateSessionStatus), OperationError> {
         spanned!("idm::server::credupdatesession<Init>", {
             let account = self.validate_init_credential_update(event.target, &event.ident)?;
-
             // ==== AUTHORISATION CHECKED ===
-
+            // #860 we have access to the qs here
```

Comment:
We can pass in the values we need to this function preferrabley over accessing the qs.